changelog.py

```diff
@@ -131,7 +131,7 @@ def collect_changelog(repo: Repository, milestone: Milestone):
         issue_text = re.sub(comment_pattern, "", issue.body).replace("\r\n", "\n")
         result = re.search(changelog_description_pattern, issue_text)
         if result is not None:
-            description = result.group("description").strip()
+            description = result.group("description").strip().strip(".")
```

Comment:
This actually means "remove `.` from the start and the end of the string", but sho would start a description with a dot? :thinking:

Reply:
You can use `rstrip` method to ignore the leading characters

Reply:
@vlad20012 ping

Reply:
Replaced to `rstrip`